pkg/filters/data.go

```diff
@@ -21,39 +20,31 @@ var _ utils.Cloner[*DataFilter] = &DataFilter{}
 
 func NewDataFilter() *DataFilter {
 	return &DataFilter{
-		filters: map[events.ID]map[string]Filter[*StringFilter]{},
+		filters: map[string]Filter[*StringFilter]{},
 		enabled: false,
 	}
 }
 
-// GetEventFilters returns the data filters map for a specific event
+// GetFieldFilters returns the data filters map for a specific event
```

Comment:
since GetFieldFilters doesn't accept events.ID anymore, should we remove the comment part "for a specific event"?